scripts/slack/notify_failure.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+import sys
+import requests
+import json
+import os
+
+def notifyslack( branch_name ):
```

Comment:
I think it might be nicer if all the scripts are consistent in exposing a function, even if right now some of them are only called directly

Reply:
Good point 👍